src/common/mmo.h

```diff
@@ -64,7 +64,8 @@
 // 20120307 - 2012-03-07aRagexeRE+ - 0x970
 
 #ifndef PACKETVER
-	#define PACKETVER 20190530
+	// from tag vXXXXXXXX.Y -> XXXXXXXXY
+	#define PACKETVER 201905300
```

Comment:
are you sure about this ?
isn't this change extremely drastic ?

for example the `<ITEMLINK>` function
https://github.com/HerculesWS/Hercules/blob/stable/npc/other/Global_Functions.txt#L484-L489
or all the `#if PACKETVER >= 20080102` `#endif` inside clif.c, all these should adds an extra digit ?

Reply:
ops, i changed wrong constant, sorry

Reply:
now correct constant used